ThePensionsRegulator.Frontend/Styles/_tpr-button.scss

```diff
@@ -19,14 +19,38 @@ $tpr-button-padding: 20px;
     border-radius: 8px;
     box-shadow: none;
     background: $govuk-button-background-colour url(/_content/ThePensionsRegulator.Frontend/tpr/chevron_right_white_24dp.svg) right center no-repeat;
+    padding: 8px 10px 7px;
     padding-left: $tpr-button-padding;
     padding-right: $tpr-button-padding *2;
 }
+input[type="file"] {
+    font-size: 1rem;
+    outline: 3px solid transparent;
+    border-color: transparent;
+    box-shadow: inset 0 0 0 2px transparent;
+    padding: 3px 3px 1px 3px;
+    font-family: $govuk-font-family;
```

Comment:
You also need to set `color: $govuk-text-colour` here as the "No file chosen" text is too dark. You can see that it's not the same as the text above it:

![image](https://github.com/user-attachments/assets/31f39a07-daf1-4e32-8cc1-3c8cea61c76b)